core/src/main/scala/com/cloudera/spark/CGroupsMemoryHandle.scala

```diff
@@ -0,0 +1,37 @@
+package com.cloudera.spark
+
+import scala.io.Source
+
+class CGroupsMemoryHandle(cgroupMemStats: Seq[(String, Long)]) extends MemoryGetter {
+
+  override val namesAndReporting: Seq[(String, PeakReporting)] = cgroupMemStats.map(x => {
+    (x._1, IncrementBytes)
+  })
+
+  override def values(dest: Array[Long], offset: Int): Unit = {
+    var counter = 0
+    cgroupMemStats.foreach(x => {
```

Comment:
same here on style.  also this is optional, but it might be slightly more readable if you use pattern matching to give names to the parts of the tuple

```scala
.foreach { case (_, metricValue) =>
  dest(offset + counter) = metricValue
  counter += 1
}
```

Reply:
Fixed in the latest commit